doc/classes/VFlowContainer.xml

```diff
@@ -1,12 +1,13 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VFlowContainer" inherits="FlowContainer" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Vertical flow container.
+		A container that arranges its child controls vertically and wraps them around at the borders.
 	</brief_description>
 	<description>
-		Vertical version of [FlowContainer].
+		A container that arranges its child controls vertically and wraps them around at the borders. This is similar to how text in a book wraps around when no more words can fit on a line.
```

Comment:
idk if *vertical layout* is really similar to a book.

Reply:
I know, it sounds like copy paste, but I actually thought hard about whether to keep this. Sure, words aren't vertical, but everyone knows how words in a book are stacked and I can't think of any nearly-as-good vertical equivalents.

Reply:
```suggestion
		A container that arranges its child controls vertically and wraps them around at the borders. This is similar to how text in a book wraps around when no more words can fit on a line, except vertically.
```
Fixed 😎

Reply:
OH. Or just "lines in a book wrapping to a new page" hehe. The answer was staring at me in the eyes.